Jenkinsfile

```diff
@@ -38,7 +38,7 @@
  */
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@bmurrell/dont-install-fuse") _
```

Comment:
I assume this is just for testing?

Reply:
Ah, yes.  Need to land that and remove it.  Thanks for noticing.